### ISSUE TITLE
ListItem: "Failed prop type: Invalid prop `Component` supplied to `PadView`."

### ISSUE THREAD

**Issue by jvandenaardweg** [react-native-training/react-native-elements#2085]
**Explain what you did**
Running my app as usual after upgrading from react-native 0.60.5 to 0.61.1

**Expected behaviour**
Should not throw a warning/error

**Describe the bug**
`ListItem` throws a warning/error. It did not do that on react-native 0.60.5

It does seem to render just fine, it just gives an warning/error in the console.

**To Reproduce**
Just render some `ListItem`'s. I get the error when rendering multiple basic `<ListItem title="test" />`

**Possible Solution**
Don't know...

**Screenshots**
<img width="517" alt="Screenshot 2019-09-25 at 18 56 04" src="https://user-images.githubusercontent.com/5155373/65622706-8b97a280-dfc6-11e9-8f98-4bf2c64e6f71.png">

Sorry for the small text, but it was the only way to capture the whole error/warning

| software              | version |
| --------------------- | ------- |
| react-native-elements | 1.2.1        |
| react-native          | 0.61.1        |
| npm           |  6.8.0       |

**Comment by betoharres** [comment]
I've just updated to react-native 0.61.1 too and got this warning as well :(

**Comment by jvandenaardweg** [comment]
Did some digging, i think this is the problem:

node_modules/react-native-elements/src/list/ListItem.js

```jsx
<PadView
  Component={ViewComponent}
  {...linearGradientProps}
  ...
```

`typeof ViewComponent` is `string`

But the proptypes expect a `func` or `object`:

```javascript
PadView.propTypes = {
  children: PropTypes.node,
  pad: PropTypes.number,
  Component: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
};
```

So, adding `PropTypes.string` makes the error go away:

```javascript
PadView.propTypes = {
  children: PropTypes.node,
  pad: PropTypes.number,
  Component: PropTypes.oneOfType([PropTypes.func, PropTypes.object, PropTypes.string]),
};
```

But I wonder if that's the solution

**Comment by betoharres** [comment]
@jvandenaardweg  Did you also experienced his error when tapping the ListItem multiple times?
```
Attempted to transition from state `RESPONDER_INACTIVE_PRESS_IN` to `RESPONDER_ACTIVE_LONG_PRESS_IN`, which is not supported. This is most likely due to `Touchable.longPressDelayTimeout` not being cancelled.
```

**Comment by jvandenaardweg** [comment]
@betoharres no, i don't have that problem

**Comment by iRoachie** [comment]
Confirmed. Looks like the types changed in react-native 0.61.0, doing a patch release in a few moments

**Comment by iRoachie** [comment]
Fixed in [v1.2.2](https://github.com/react-native-training/react-native-elements/releases/tag/v1.2.2). Thanks to everyone for reporting so quickly.

**Comment by betoharres** [comment]
Thank you so much @iRoachie for the quick update! This is why I love this library!
